parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -134,6 +134,11 @@ installMoby() {
     fi
 }
 
+installContainerd() {
+	installMoby
```

Comment:
we already call `installMoby` in `installContainerRuntime` at https://github.com/Azure/aks-engine/blob/master/parts/k8s/cloud-init/artifacts/cse_install.sh#L112

It might make sense to change the condition for that call from 
```
    if [[ "$CONTAINER_RUNTIME" == "docker" ]]; then
        installMoby
    fi
```
to
```
    if [[ "$CONTAINER_RUNTIME" == "docker" || "$CONTAINER_RUNTIME" == "kata-containers" || "$CONTAINER_RUNTIME" == "containerd" ]]; then
        installMoby
    fi
```
To have all calls of `installMoby` in one place and make the code easier to read. Otherwise it's not clear what the intentions are in terms of when to install Moby.